HaaLimits/python/HaaLimits.py

```diff
@@ -33,8 +33,8 @@ class HaaLimits(Limits):
     XLABEL = 'm_{#mu#mu}'
     UPSILONRANGE = [7, 12]
 
-    REGIONS = ['FP','PP']
-    SHIFTS = ['Pileup', 'ID','Iso', 'BTag']
+    REGIONS = []
```

Comment:
Dont' change the regions, just the shifts
The analysis doesnt work if you dont have these regions.